packages/api/scripts/cli.js

```diff
@@ -32,20 +33,23 @@ prog
       outfile: path.join(__dirname, '..', 'dist', 'worker.js'),
       legalComments: 'external',
       inject: [path.join(__dirname, 'node-globals.js')],
-      plugins: [{
-        name: 'alias',
-        setup (build) {
-          build.onResolve({ filter: /^stream$/ }, () => {
-            return { path: require.resolve('stream-browserify') }
-          })
-          build.onResolve({ filter: /^node-fetch$/ }, () => {
-            return { path: path.resolve(__dirname, 'fetch.js') }
-          })
-          build.onResolve({ filter: /^cross-fetch$/ }, () => {
-            return { path: path.resolve(__dirname, 'fetch.js') }
-          })
-        }
-      }],
+      plugins: [
+        {
+          name: 'alias',
+          setup (build) {
+            build.onResolve({ filter: /^stream$/ }, () => {
+              return { path: require.resolve('stream-browserify') }
+            })
+            build.onResolve({ filter: /^node-fetch$/ }, () => {
+              return { path: path.resolve(__dirname, 'fetch.js') }
+            })
+            build.onResolve({ filter: /^cross-fetch$/ }, () => {
+              return { path: path.resolve(__dirname, 'fetch.js') }
+            })
+          }
+        },
+        NodeModulesPolyfillPlugin()
```

Comment:
Added a comment explaining how 'stripe' deps on some node things (though I think we dont actually import those parts, e.g. we use the Fetch http adapater instead of the Node one)

Reply:
(afaict it wasn't related to the use of `assert`)